docs/gallery/decision_support/2_supply_chain/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Supply Chain Optimization
+category: decision_support
+data-keywords: optimization scenario cycle comparison
```

Comment:
Everything looks very good! Just missing `enterprise` keyword here

Reply:
Yes, I didn't commit it. Done